sss/tests/test_puma.py

```diff
@@ -45,6 +46,17 @@ def test_read_puma():
 def test_puma_crosswalk():
     """Test creating the crosswalk for WA."""
     year = 2015
+
+    with pytest.raises(
+        ValueError, match="Must pass the nyc_wa_path if processing NY or WA"
+    ) as exc_info:
+        puma_crosswalk(
+            os.path.join(DATA_PATH, "puma_data", "puma_text", "PUMSEQ10_53.txt"), year
+        )
+
+    assert exc_info.type is ValueError
+    assert exc_info.value.args[0] == "Must pass the nyc_wa_path if processing NY or WA"
```

Comment:
Same comment here, you don't need these asserts, they're guaranteed by the `pytest.raises` call.